keras/callbacks.py

```diff
@@ -2716,6 +2716,11 @@ def on_train_batch_end(self, batch, logs=None):
                 1.0 / batch_run_time,
                 step=self._train_step,
             )
+
+        if logs:
+            for name, value in logs.items():
```

Comment:
Thank you for you PR! I suspect this may not work with `ParameterServerStrategy`: can you check?

Reply:
I couldn't figure out how to get `ParameterServerStrategy` running, sorry about that. If this is an acceptable workaround, I turned off this block by default so that any errors (if they exist) should only show up if `ParameterServerStrategy` and `update_freq` are being used at the same time.

Reply:
[This](https://github.com/keras-team/keras/blob/master/keras/integration_test/distributed_training_test.py) and [this](https://github.com/keras-team/keras/blob/master/keras/integration_test/parameter_server_custom_training_loop_test.py) are two tests you can refer to to create a locally-run unit test that uses `ParameterServerStrategy`. Mind checking with that?